lib/src/Uncertainty/Distribution/openturns/JointDistribution.hxx

```diff
@@ -71,12 +71,15 @@ public:
   void setDistributionCollection(const DistributionCollection & coll);
   DistributionCollection getDistributionCollection() const;
 
+  /** Kernel accessor */
+  void setKernel(const Distribution & kernel);
+  Distribution getKernel() const;
+
   /** Copula accessor */
   void setCopula(const Distribution & copula);
   Distribution getCopula() const override;
```

Comment:
so setCopula should be deprecated ?

Reply:
I don't think so. A copula is a valid kernel and is more explicit when one want to explicitely use this class to build a joint distribution based on Sklar's theorem.
Another option is to use this class as a base class for others, one based on copulas, one based on uniform order statistics and so one.